src/actions/lookupObject.ts

```diff
@@ -0,0 +1,93 @@
+import Client from '../client';
+
+const { messages } = require('elasticio-node');
+
+// eslint-disable-next-line max-len
+exports.process = async function ProcessAction(msg: { body: { lookupCriteriaValue: any; }; }, cfg: { allowIdToBeOmitted?: any; baseUrl?: any; allowZeroResults?: any; objectType?: any; lookupCriteria?: any; }) {
```

Comment:
You may use `async function ProcessAction(msg: any, cfg: any)`
And if you want to define parameters structure - pls use it as variable

Reply:
done